### PR TITLE
Add navbar search button

### LINKED ISSUE #3: Implement Nav
- [ ] Desktop version
- [ ] Mobile version

### PR DESCRIPTION
**Pull request by jose-correia**
This PR adds the search button to the navigation bar, and cleans some unnecessary CSS classes.

The button does not have any functionality yet.

Fix #3

Test URLs:
- Before: https://main--moleculardevices--hlxsites.hlx.page/
- After: https://navbar-add-search-button--moleculardevices--hlxsites.hlx.page/

### CODE REVIEW

File: styles/styles.css [hunk 1]
```diff
@@ -184,6 +184,19 @@ main.text-caption {
   color: var(--text-gray);
 }
 
+.fa {
```
Comment (mhaack) [inline styles/styles.css:187]: This should not be needed. We have https://github.com/hlxsites/moleculardevices/blob/main/styles/font-awesome.min.css added already.

Comment (jose-correia) [reply styles/styles.css]: You're right! changed

File: blocks/header/header.css [hunk 2]
```diff
@@ -248,6 +230,64 @@ header nav .nav-tools {
...
+    display: block;
+    padding: 15px 0 15px 13px;
+    position: relative;
+  }
+
+  .nav-tabs > li.fa-search::before {
+    background-color: #fff;
+    padding: 9px;
+    border-radius: 50%;
+    border: 2px solid #ee7624;
```
Comment (mhaack) [inline blocks/header/header.css:249]: Is this a common color? The we should have a variable for it.

File: blocks/header/header.css [hunk 3]
```diff
@@ -248,6 +230,64 @@ header nav .nav-tools {
     font-size: 14px;
   }
 
+  .nav-tabs > li.header-search {
+    margin-top: 0;
+    text-align: center;
+  }
+
+  .nav-tabs > li.fa-search {
+    color: #34393d;
```
Comment (mhaack) [inline blocks/header/header.css:239]: s this a common color? The we should have a variable for it.

File: blocks/header/header.css [hunk 4]
```diff
@@ -248,6 +230,64 @@ header nav .nav-tools {
...
+    border-radius: 100%;
+    height: 161px;
+    width: 135px;
+    margin-top: -99px;
+    display: flex;
+    align-items: flex-end;
+    box-shadow: -8px 4px rgb(238 118 36 / 30%);
+    z-index: 9999 !important;
+    position: relative;
+    background: #ee7624;
```
Comment (mhaack) [inline blocks/header/header.css:277]: Is this a common color? The we should have a variable for it.

File: blocks/header/header.js [hunk 5]
```diff
@@ -19,11 +19,18 @@ function buildTools(content) {
 
 function buildRequestQuote() {
   const requestQuote = document.createElement('li');
-  requestQuote.innerHTML = `<li class="we-mega-menu-li header-rfq">
-  <a class="we-mega-menu-li" title="" href="/quote-request?cid=12" target="">Request<br>Quote</a></li>`;
+  requestQuote.classList.add('we-menu-li', 'header-rfq');
```
Comment (mhaack) [inline blocks/header/header.js:22]: Cleanup classnames please.

File: blocks/header/header.js [hunk 6]
```diff
@@ -19,11 +19,18 @@ function buildTools(content) {
...
+  requestQuote.innerHTML = '<a class="we-mega-menu-li" title="" href="/quote-request?cid=12" target="">Request<br>Quote</a>';
   return requestQuote;
 }
 
+function buildSearch() {
+  const search = document.createElement('li');
+  search.classList.add('we-menu-li', 'searchlink', 'header-search', 'fa', 'fa-search');
```
Comment (mhaack) [inline blocks/header/header.js:29]: Cleanup classnames please, why do we need `we-menu-li`

File: blocks/header/header.js [hunk 7]
```diff
@@ -19,11 +19,18 @@ function buildTools(content) {
...
+  search.innerHTML = '<a class="we-mega-menu-li" title="" href="#" target="">Search</a>';
```
Comment (mhaack) [inline blocks/header/header.js:30]: Why do we need `we-mega-menu-li`

**Comment by mhaack** [comment]
Not exactly related to this PR but why do we need the nested `div` here?

Screenshot 2023-03-15 at 16 09 23

**Comment by jose-correia** [comment]
> Not exactly related to this PR but why do we need the nested `div` here?
> 
> Screenshot 2023-03-15 at 16 09 23

@mhaack we don't! I was simply grabbing the list item that comes from the doc. I now added logic to remove the nested div inside

File: blocks/header/header.css [hunk 8]
```diff
@@ -12,6 +12,17 @@ header .block {
   padding-bottom: 46px;
 }
 
+.nav-tabs li {
```
Comment (mhaack) [inline blocks/header/header.css:15]: We have some classes defined with `header` prefix and some not. We keep that consistant.

**Review by mhaack — APPROVED** [review]

File: blocks/header/header.css (hunk 8 above)
Comment (jose-correia) [reply blocks/header/header.css:15]: 👍

**Pull request merged**